### PR TITLE
Add documentation for Wazuh as a custom source ASL integration

### PR DESCRIPTION
**Pull request by AlexRuiz7**
### Docs building
- [x] Compiles without warnings.
### Code formatting and web optimization
- [x] Uses three spaces indentation.
- [x] Adds or updates meta descriptions accordingly.
- [x] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [ ] Uses present tense, active voice, and semi-formal registry.
- [ ] Uses short, simple sentences.
- [ ] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

**Review by f-galland — DISMISSED** [review]

File: source/cloud-security/amazon/services/supported-services/amazon-security-lake/security-lake-source.rst [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+    .. code-block:: console
+
+        sudo systemctl enable logstash
+        sudo systemctl start logstash
+
+
+OCSF Mapping
+-------------
+
+The integration maps Wazuh Security Events to the **OCSF v1.1.0** `Security Finding (2001) <https://schema.ocsf.io/classes/security_finding>`__ Class.
```
Comment (javimed) [inline source/cloud-security/amazon/services/supported-services/amazon-security-lake/security-lake-source.rst:246]: Use italics for for emphasis. For example `*OCSF v1.1.0*`. Use bold font style for UI elements.

File: source/cloud-security/amazon/services/supported-services/amazon-security-lake/security-lake-source.rst
Comment (javimed) [inline source/cloud-security/amazon/services/supported-services/amazon-security-lake/security-lake-source.rst:1]: Consider using code font style in tables for attribute names, attribute values, class names, data types, etc. For example ` ``200101`` `, ` ``type_uid`` `, ` ``Wazuh, Inc.`` `, ` ``time`` ` (if we mean a key, field, etc. named "time")

Comment (javimed) [inline source/cloud-security/amazon/services/supported-services/amazon-security-lake/security-lake-source.rst:1]: `wazuh-indexer` is the service name, if we mean the component we say `the Wazuh indexer`.

Comment (javimed) [inline source/cloud-security/amazon/services/supported-services/amazon-security-lake/security-lake-source.rst:1]: You can specify the prompt symbol. For example

```
   .. code-block:: console

      $ sudo /usr/share/logstash/bin/logstash-plugin install logstash-input-opensearch
```

Copy-to-clipboard discards leading `$` and `#` so no worries about that.

**Review by javimed — CHANGES_REQUESTED** [review]

**Pull request merged**